src/components/userFeedback/UserFeedback.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import { shallow, ShallowWrapper } from 'enzyme';
+import { UserFeedback, UserFeedbackState, IUserFeedbackProps, TextColorClasses } from './UserFeedback';
+
+// Until Webpack provided plugins works with TS 2.0
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+describe('<UserFeedback>', () => {
+  const defaultTextColorClass: string = TextColorClasses.default;
+  const errorTextColorClass: string = TextColorClasses.error;
+
+  const generateProps = (feedbackText: string, state: string, extraClasses?: string, displayOnShow?: string): IUserFeedbackProps => {
+    return displayOnShow ? { feedbackText, state, extraClasses, displayOnShow } : { feedbackText, state, extraClasses };
+  };
+
+  const defaultProps: IUserFeedbackProps = generateProps('', '', '');
```

Comment:
This const is only used once. I would simply call generateProps() like you do in other tests cases.